### PR TITLE
finetuning connect-popup tests

### PR DESCRIPTION
**Pull request by mroz22**
couple of changes for webextension tests:

- simplify popup-close test a bit.
- make some of the flakier tests run only in nightly

### CODE REVIEW

**Review by martykan — APPROVED** [review]

**Review by karliatto — APPROVED** [review]
Simplification is always good. :+1:

File: packages/connect-popup/e2e/tests/popup-close.test.ts [hunk 1]
```diff
@@ -48,529 +37,380 @@ test.beforeAll(async () => {
...
+    );
+
+    if (isNextra) {
+        await page.click("a[data-test='@navbar-logo']");
+        await page.click("a[href$='/methods/bitcoin/verifyMessage/']");
+    } else {
+        await explorerPage.goto(`${explorerUrl}#/method/verifyMessage`);
+    }
+
+    await explorerPage.pause();
```
Comment (martykan) [inline packages/connect-popup/e2e/tests/popup-close.test.ts]: So this is why Nextra CI freezes 😄

**Review by martykan — CHANGES_REQUESTED** [review]

File: packages/connect-popup/e2e/tests/popup-close.test.ts (hunk 1 above)
Comment (mroz22) [reply packages/connect-popup/e2e/tests/popup-close.test.ts]: aaargh

File: packages/connect-popup/e2e/tests/popup-close.test.ts [hunk 2]
```diff
@@ -48,529 +37,378 @@ test.beforeAll(async () => {
...
+    const hasLogs = await checkHasLogs(logPage);
+    log(`hasLogs: ${hasLogs}`);
+    if (hasLogs) {
+        log('afterEach', 'downloading logs');
+        await downloadLogs(logPage, `./test-results/log-${addDashesToSpaces(testInfo.title)}.txt`);
+    } else {
+        log('afterEach', 'no logs');
+    }
+
+    await logPage.pause();
```
Comment (mroz22) [inline packages/connect-popup/e2e/tests/popup-close.test.ts]: there is one more :P

Comment (martykan) [reply packages/connect-popup/e2e/tests/popup-close.test.ts]: Ok now let's see

**Review by martykan — APPROVED** [review]
Some Nextra stuff is failing but I'll fix that separately

**Pull request merged**